src/gpu/nvidia/cudnn_convolution_impl.hpp

```diff
@@ -676,7 +676,7 @@ struct cudnn_convolution_impl_fwd_t : public cudnn_convolution_impl_base_t {
                     continue;
                 fwd_alg_kind = perf[i].algo;
                 CHECK(CUDNN_EXECUTE_FUNC_S(cudnnSetConvolutionMathType,
-                        conv_desc, perf[i].mathType));
```

Comment:
Can you please elaborate on this change?

Reply:
We observed that by default it was taking `CUDNN_TENSOR_OP_MATH` for bf16 which was returning `nan` values. So to fix that we made the above change.